location-services/read-data.qmd

```diff
@@ -222,3 +222,47 @@ fl_query |>
   collect()
 ```
 
+## Map and Feature Servers
+
+This example has only illustrated how to work with `FeatureLayer` objects. However, often times you may wish to work with a collection of layers in either a `FeatureServer` or `MapServer`. Both of these are collections of multiple layers. Like a `FeatureLayer`, these are accessed with `arc_open()`.
```

Comment:
```suggestion
This example has only illustrated how to work with `FeatureLayer` objects. However, often times you may wish to work with a collection of layers in a `FeatureServer`, `MapServer`, or `GroupLayer`. All of these are collections of multiple layers. Like a `FeatureLayer`, these are accessed with `arc_open()`.
```
I tested and this also works with a GroupLayer?! super cool. It's a little confusing because a GroupLayer is part of a MapServer so it ends up being an extra nested FeatureLayer situation, but hopefully this would be clear enough for people to figure it out?